internal/common/windows_utils.bzl

```diff
@@ -96,7 +96,12 @@ set RUNFILES_MANIFEST_ONLY=1
 call :rlocation "{sh_script}" run_script
 for %%a in ("{bash_bin}") do set "bash_bin_dir=%%~dpa"
 set PATH=%bash_bin_dir%;%PATH%
-"{bash_bin}" -c "!run_script! %*"
+set args=%*
```

Comment:
would be nice to have some comments here to explain that this is escaping the arguments

Reply:
Cool, comments added.